source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+| days_post_transplant    | the number of      | numeric         | 7, 14, 21, 49    |
+|                         | days after the     |                 |                  |
+|                         | mice were          |                 |                  |
+|                         | humanized          |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+
+
+Even though the mouse ID looks like a number, we will specify the type using the ``#q2_type`` column in the data.
+
+The metadata is avaliable as a `Google Sheet`_, or ou can download it directly from and save it as a tsv.
+
+.. download::
+   :url: https://data.qiime2.org/2019.7/tutorials/pd-mice/sample_metadata.tsv
+   :saveas: metadata.tsv
+
+The sample metadata will be used through out the tutorial.
+
+Loading the data into QIIME 2
+=============================
+
+In QIIME 2, all data is structured as an Artifact of a specific semantic type. The artifacts contain the data as well as information about the data, including a record of the original data, the tools used to process it. This allows for better tracking of how you actually got to where you are in your analysis. You can learn more about common QIIME 2 Artifacts and types of artifacts :doc:`here <../semantic-types/>`.
+
+Our samples were amplified u|sing the `EMP 515f-806r`_ primers and sequenced on an Illumina MiSeq with a 2x150bp kit. The hypervariable
+region covered by the primers we used 290bp and so with 150bp reads, our sequences will be slightly too short to be able to do paired-end analysis downstream. Therefore, we’re going to work with single-end sequences. We will work with a version of the samples which have already been demultiplexed, for example, by the sequencing center. If you need to demultiplex your sequences, the doc: `moving pictures tutorial <moving-pictures>` describes how to demultiplex your sequences if they were sequenced using the Earth Microbiome Project protocol.
+
+We will load the sequences as ``SampleData[SequencesWithQuality]``, which is the single end sequence demultiplexed format. If we wanted to import paired sequences, we would chose the ``SampleData[PairedEndSequencesWithQuality]`` type. We will import the sequences using the sample manifest format. This is one of the most versatile ways to import demultiplexed data in QIIME 2. We create a tab-separated sample manifest file that maps the sample name we want to use in QIIME 2 to the path to the sequence file, and the read direction. The benefit is that the demultiplexed sequence files can be named anything you want; there are not fixed assumptions about the conventions, and the file names do not dictate the final name. When QIIME 2 reads the file, it ignores any line prefixed with the ``#`` symbol. The first line that doesn’t contain a ``#`` is the header line and must be ``sample-id\tabsolute-filepath``. The sample order after the header line does not matter.
```

Comment:
```suggestion
We will import the sequences as ``SampleData[SequencesWithQuality]``, which is the single end sequence demultiplexed format. If we wanted to import paired sequences, we would choose the ``SampleData[PairedEndSequencesWithQuality]`` type. We will import the sequences using the sample manifest format. This is one of the most versatile ways to import demultiplexed data in QIIME 2. We create a tab-separated sample manifest file that maps the sample name we want to use in QIIME 2 to the path to the sequence file. The benefit is that the demultiplexed sequence files can be named anything you want; there are not fixed assumptions about the conventions, and the file names do not dictate the final name. When QIIME 2 reads the file, it ignores any line prefixed with the ``#`` symbol. The first line that doesn’t contain a ``#`` is the header line and must be ``sample-id\tabsolute-filepath``. The sample order after the header line does not matter.
```

I wonder if we should just add this column to the sample metadata file and call it a day. Thoughts @jwdebelius & @nbokulich?

Reply:
I think my experience with Qiita is one reason for seperation. But I'm not wed to the idea... just means updating the mapping file and data dictionary